botocore/data/elb/2012-06-01/waiters-2.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version":2,
+    "waiters":{
+        "InstanceDeregistered": {
```

Comment:
Yeah, I think we should hold off on this for now until we can look into this waiter more in depth.